README.md

```diff
@@ -46,3 +46,24 @@ If you are building for 2.5.x you can re-use the 2.4.x build.
 1. As root, run: `service nn-analytics start`
 2. (Optional) On your browser hit: `http(s)://HOSTNAME:8080/namespace.html` and fetch the latest image and then `service nn-analytics restart` if this is your first run.
 3. If any issues occur check `/var/log/nn-analytics` for logs.
+
+## Additional License Info:
+```
+NAMENODE ANALYTICS SUBCOMPONENTS:
```

Comment:
this should go in LICENSE file.

Reply:
According to GitHub info here: https://help.github.com/articles/adding-a-license-to-a-repository/, additional complexities to the LICENSE can be moved to the README.md file.

Check here: https://help.github.com/articles/licensing-a-repository/
Specifically: 
```
If your repository is using a license that is listed on the Choose a License website and it's not displaying clearly at the top of the repository page, it may contain multiple licenses or other complexity. To have your license detected, simplify your LICENSE file and note the complexity somewhere else, such as your repository's README file.
```

Reply:
ok, since you can, +1